plugins/Installation/FormFirstWebsiteSetup.php

```diff
@@ -37,6 +39,13 @@ function init()
         $timezones = API::getInstance()->getTimezonesList();
         $timezones = array_merge(array('No timezone' => Piwik::translate('SitesManager_SelectACity')), $timezones);
 
+        // Use server timezone as default. If server timezone is UTC, it is likely
+        // a default not specified explicitly by the sysadm, so ignore this.
+        $timezone = PIWIK_DEFAULT_TIMEZONE;
+        if (in_array($timezone, array('UTC', 'Etc/UTC', 'GMT', 'Etc/GMT'))) {
+            $timezone = null;
+        }
```

Comment:
I wonder if we need to check whether Matomo "knows" the timezone (present in `$timezones`)?

Reply:
The value determines the default selected timezone in the select field. If an invalid timezone is submitted, QuickForm will just ignore it, so there is no need for further validation.